src/components/layout/nav/search/Search.jsx

```diff
@@ -1,7 +1,151 @@
+    images: [
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332979914_01_a?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332654173_01_a-Nike-Mens-Flex-Control-3-Training-Shoes-Black-Anthracite-AJ5911-002?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332762662_10_a-Nike-Womens-Air-Precision-II-Basketball-Shoes-White-Black-AA7069-100?bgColor=0,0,0,0&fmt=png-alpha&hei=528&resMode=sharp&qlt=85,1&op_sharpen=1",
+    ],
+  },
+  {
+    title: "YEEZY BOOST 350",
+    brand: "Adidas",
+    price: 399,
+    currency: "$",
+    tags: ["shoe", "shoes", "adidas", "yeezy", "boost"],
+    images: [
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332979914_01_a?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332654173_01_a-Nike-Mens-Flex-Control-3-Training-Shoes-Black-Anthracite-AJ5911-002?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332762662_10_a-Nike-Womens-Air-Precision-II-Basketball-Shoes-White-Black-AA7069-100?bgColor=0,0,0,0&fmt=png-alpha&hei=528&resMode=sharp&qlt=85,1&op_sharpen=1",
+    ],
+  },
+  {
+    title: "V2 ZYON",
+    brand: "Adidas",
+    price: 250,
+    currency: "$",
+    tags: ["shoe", "shoes", "adidas", "v2", "zyon"],
+    images: [
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332979914_01_a?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332654173_01_a-Nike-Mens-Flex-Control-3-Training-Shoes-Black-Anthracite-AJ5911-002?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332762662_10_a-Nike-Womens-Air-Precision-II-Basketball-Shoes-White-Black-AA7069-100?bgColor=0,0,0,0&fmt=png-alpha&hei=528&resMode=sharp&qlt=85,1&op_sharpen=1",
+    ],
+  },
+];
 
 const Search = () => {
-  return <div></div>;
+  // It is left empty because we want search to start as empty.
+  const [searchQueryContainer, setSearchQueryContainer] = useState("");
+  const [showEmptySearchWarning, setShowEmptySearchWarning] = useState(false);
+
+  const ONLY_FIRST_IMAGE_OF_PRODUCTS = 0;
+  const AMOUNT_OF_PRODUCT_IN_SEARCH_PREVIEW = 3;
+
+  // TODO(emrerdem1): Link this to database.
+  // This will be sending input value to database with fetching if input is not empty when "Search" button is clicked.
+  const sendSearchQuery = () => {
+    isSearchInputEmpty();
+  };
+
+  // Set input values by onChange and show preview card only if there is something typed.
+  const handleSearchChanges = (e) => {
+    setSearchQueryContainer(e.target.value);
+    isSearchInputEmpty();
+  };
+
+  const isSearchInputEmpty = () =>
```

Comment:
this variable name is wrong. isX suggests it's a boolean, but actually this function updates something